### ISSUE TITLE
Do you ever wish to go back to an individual contributor role?

### ISSUE THREAD

**Issue by JohnPhamous** [jasoncwarner/ama#30]

**Comment by jasoncwarner** [comment]
Yes, sometimes. There is a very specific satisfaction that comes from programming and working on a team in that way. 

And for me, the 0 to 1 creation aspect of software has a special thrill and gives a kind of rush you don't have when you aren't developing. And it's not the same to see someone else create something even if you are directing it. 

Related to sports. 

I like sports, but I like *playing* sports. I also like coaching or GMing sports. In fact, I am a much better coach or executive even in sports than I ever was a player. But playing is where the fun happens. 

It's a bit of the same in business. 

However, here's the real answer. 

I was an average developer. Just like in sports where I had some skill but wasn't the most talented or gifted athlete (I had to work HARD to get everything I had in sports). I am far more skilled an executive than I ever was a developer and there is an entirely different type of satisfaction I get out of being really really good at my job. In that way, I find it very fulfilling to keep my work as executive management and my hobby as programming. It feels like a good compromise to maximize what I *personally* need out of both. 

If I'm honest with myself about sports, I would spend my 'professional' time as a coach/GM and get my personal satisfaction out of practicing my 3 point shot. Would be the same compromise.
